files/zh-cn/web/javascript/reference/functions/arguments/callee/index.html

```diff
@@ -17,15 +17,15 @@
 
 <h2 id="Description">描述</h2>
 
-<p><strong><code>callee</code></strong> 是 <code>arguments</code> 对象的一个属性。它可以用于引用该函数的函数体内当前正在执行的函数。这在函数的名称是未知时很有用，例如在没有名称的函数表达式 (也称为“匿名函数”)内。</p>
+<p><strong><code>callee</code></strong> 是 <code>arguments</code> 对象的一个属性。它可以用于引用该函数的函数体内当前正在执行的函数。这在函数的名称是未知时很有用，例如在没有名称的函数表达式 (也称为“匿名函数”) 内。</p>
 
-<div class="warning"><p><strong>警告：</strong>在<a href="/en-US/docs/JavaScript/Reference/Functions_and_function_scope/Strict_mode" title="/en-US/docs/JavaScript/Reference/Functions_and_function_scope/Strict_mode">严格模式</a>下，第5版 ECMAScript (<strong>ES5</strong>) 禁止使用 <code>a<strong>rguments.callee()</strong>。当一个函数必须调用自身的时候, 避免使用 <strong>arguments.callee(), </strong></code>通过<code>要么</code>给函数表达式一个名字,要么使用一个函数声明.</p></div>
+<div class="warning"><p><strong>警告：</strong>在<a href="/en-US/docs/JavaScript/Reference/Functions_and_function_scope/Strict_mode" title="/en-US/docs/JavaScript/Reference/Functions_and_function_scope/Strict_mode">严格模式</a>下，第 5 版 ECMAScript (<strong>ES5</strong>) 禁止使用 <code>a<strong>rguments.callee()</strong>。当一个函数必须调用自身的时候，避免使用 <strong>arguments.callee(), </strong></code>通过<code>要么</code>给函数表达式一个名字，要么使用一个函数声明。</p></div>
 
-<h2 id="为什么_arguments.callee_从ES5严格模式中删除了？">为什么 arguments.callee 从ES5严格模式中删除了？</h2>
+<h2 id="为什么_arguments.callee_从ES5严格模式中删除了？">为什么 arguments.callee 从 ES5 严格模式中删除了？</h2>
```

Comment:
@irvin do you think we should update the `id` like this (it's used in MDN markdown):

```suggestion
<h2 id="为什么_arguments.callee_从_es5_严格模式中删除了？">为什么 arguments.callee 从 ES5 严格模式中删除了？</h2>
```